tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+import song2txt.txt_parser.song_line as sl
+from song2txt.txt_parser.ultrastar_txt import UltraStarTXT
+from song2txt.txt_parser.metadata import MetaData
+
+# Constants
+# Song Lines
+START_BEAT = 0
+LENGTH = 1
+MIDI = 60
+WORD = 'Word'
+PLAYER = 1
+END_BEAT = 1
+
+# TXT parsing
+PATH = 'song.txt'
+FILE_END = 'E'
```

Comment:
Si esta 'E' es la misma que la 'E' de arriba, deben usar la misma *source of truth*.